pkg/k8s/apis/cilium.io/client/cell.go

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	k8sClient "github.com/cilium/cilium/pkg/k8s/client"
+)
+
+// SkipCRDCreation specifies whether the CustomResourceDefinition will be
+// created by the daemon
+const SkipCRDCreation = "skip-crd-creation"
```

Comment:
Now that the option has been moved, the cmdref documentation needs to be rebuilt. You can do that simply running:

`make -C Documentation/ update-cmdref`